addon/components/nypr-brick-item.js

```diff
@@ -0,0 +1,27 @@
+import Component from 'ember-component';
+import computed, { reads } from 'ember-computed';
+import { equal } from 'ember-computed';
+import { htmlSafe } from 'ember-string';
+import get from 'ember-metal/get';
+import {imageTemplate} from 'nypr-ui/helpers/image-template';
+
+export default Component.extend({
+  tagName: '',
+  pk: reads('item.id'),
+  vertical: equal('template', 'vertical'),
+  backgroundImage: computed('item.attributes.imageMain.url', function() {
+    var imageMain = get(this, 'item.attributes.imageMain');
+    var urlString;
+    if (imageMain){
+      // just in case we don't get a template and crop from the API, fallback to url
+      if (imageMain.template && imageMain.crop){
+        urlString = imageTemplate([imageMain.template, 800,  0, imageMain.crop]);
+      } else {
+        urlString = imageMain.url;
+      }
+      return htmlSafe(`background-image: url(${urlString});`);
+    } else {
+      return '';
```

Comment:
would you wrap this in `htmlSafe`? otherwise deprecation warnings are printed to the console.